### PR TITLE
10 get users

### PR DESCRIPTION
**Pull request by addleton**

### CODE REVIEW

File: endpoints.json [hunk 1]
```diff
@@ -93,5 +93,18 @@
...
+        "queries": [],
+        "exampleResponse": {
+            "users": [
+                {
+                    "username": "lurker",
+                    "name": "do_nothing",
+                    "avatar_url": "https://www.golenbock.com/wp-content/uploads/2015/01/placeholder-user.png"
+                }
+            ]
+        }
```
Comment (itskatherine) [inline endpoints.json:108]: Great 👍

File: errors.js [hunk 2]
```diff
@@ -12,4 +12,7 @@ exports.handleCustomErrors = (err, req, res, next) => {
         res.status(err.status).send({ msg: err.msg });
     }
     next(err);
-};
\ No newline at end of file
+};
+exports.handleInvalidUrl = (err, req, res, next) => {
```
Comment (itskatherine) [inline errors.js:16]: This is not an error handling middleware (i.e. it shouldn't be invoked with an `err` arg) 
It's basically a controller triggered by any endpoint that you haven't specified in app
I think the 404 currently caught in your tests is due to the default express functionality

**Review by itskatherine — COMMENTED** [review]
Hi Michael, this looks good to merge
Though I don't think your invalid url endpoint is functioning currently
If it's working you would be able to in your test access the message "Not found" in your response as you've specified in line 17 of errors.js
You can either fix this or rely on the 404 bad path handling that comes by default in express

File: errors.js (hunk 2 above)
Comment (addleton) [reply errors.js:16]: Thanks, I'll take a look tomorrow. :+1:

**Pull request merged**